app/database/schemas_v2.py

```diff
@@ -0,0 +1,305 @@
+"""Pydantic schema models."""
+from datetime import datetime
+from typing import Annotated, Any, Dict
+
+from pydantic import BaseModel, EmailStr, Field, HttpUrl, SecretStr, validator
+
+from app.database import models_v2
+from app.internal.privacy import PrivacyKinds
+
+
+class EventBase(BaseModel):
+    """Base Event schema model. Should not be used directly."""
+
+    title: Annotated[
```

Comment:
I think we might want to keep supporting Python 3.8 to allow Ubuntu 20.04 LTS support our system OOTB

Reply:
Done.